frontend/src/state/actions/index.js

```diff
@@ -2,4 +2,6 @@ export * from './auth';
 export * from './invite';
 export * from './register';
 export * from './changePassword';
```

Comment:
there's actions/changePassword and actions/types/password why not consistent names?